newrelic-agent/build.gradle

```diff
@@ -69,7 +69,9 @@ dependencies {
     }
 
     shadowIntoJar("com.newrelic.agent.java:newrelic-module-util-java:2.0")
-
+//    shadowIntoJar("com.newrelic:jfr-daemon:1.1.0")
+    // FIXME change to a published artifact when available, jfr-daemon is required to configure jfr service 
+    shadowIntoJar files('/Users/xxia/dev/newrelic-jfr-core/jfr-daemon/build/libs/jfr-daemon-1.2.0-SNAPSHOT.jar')
```

Comment:
Tests will continue to fail on this PR until we've published a new version of the jfr-daemon and modified this line to use that as a dependency. Using the locally published dependency all of the tests pass. This is blocked by this PR: https://github.com/newrelic/newrelic-jfr-core/pull/178